core/src/main/java/com/graphhopper/storage/GraphStorage.java

```diff
@@ -24,8 +24,6 @@
 
     EncodingManager getEncodingManager();
 
-    void setSegmentSize(int bytes);
```

Comment:
I think this whole interface is basically not needed. There is only one single usage in `OSMReader` to call `create` and `GraphStorage` even gets the `create` method from `Storable`. All these methods here are never used via the interface.